### PR TITLE
stdevOsmel t21 list transactions

### PR DESCRIPTION
**Pull request by stdevAlDen**
The layout of the transaction list was created

### CODE REVIEW

File: src/pipes/sky-date/sky-date.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ */
+@Pipe({
+  name: 'skydate',
+})
+export class SkyDatePipe implements PipeTransform {
+  /**
+   * Takes a value and makes it transform like the design.
+   */
+  transform(value: string) {
+    let _value: string = '';
```
Comment (samuelvisscher) [inline src/pipes/sky-date/sky-date.ts:17]: dont use let for vars that are not reassigned
dont type hint when its automatically deduced
dont prefix protected vars with _, use private scope for properties tho

**Review by samuelvisscher — CHANGES_REQUESTED** [review]
Why don't we use the standard Angular date format pipe or alternatively momentjs?

**Comment by samuelvisscher** [comment]
Also, could you please make this PR against dev?

**Comment by stdevGeekmidget** [comment]
Please close this PR, we do not need it in develop branch!!

**Pull request closed without merging**